v3/spring-cloud-aws-v3-ses/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>v3</artifactId>
+		<groupId>io.awspring.cloud</groupId>
+		<version>2.3.0-RC2</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-aws-v3-ses</artifactId>
+	<name>Spring Cloud AWS V3 SimpleEmailService</name>
+	<description>Spring Cloud AWS V3 SimpleEmailService</description>
+
+	<dependencies>
+
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-context-support</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>io.awspring.cloud</groupId>
+			<artifactId>spring-cloud-aws-v3-autoconfigure</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>software.amazon.awssdk</groupId>
+			<artifactId>sesv2</artifactId>
```

Comment:
AFAIK, `sesv2` allow integration with `pinpoint` service and `ses` with `simple email service`. For now, the dependency to use would be `ses` instead.

Reply:
Actually `sesv2` interacts with the AWS SES V2 service. See [here](https://docs.aws.amazon.com/ses/latest/APIReference-V2/Welcome.html). The difference with the `ses` AWS SDK is that it uses the old `2010-12-01` schema for SES. See [here](https://sdk.amazonaws.com/java/api/latest/overview-summary.html) for the Javadoc.

<img width="925" alt="Screen Shot 2021-02-22 at 9 31 53 PM" src="https://user-images.githubusercontent.com/2806069/108734372-69a4a580-7555-11eb-9908-42a8f44bbe17.png">

`pinpoint` is entirely different SDK whose maven coordinates are:

```
<!-- https://mvnrepository.com/artifact/software.amazon.awssdk/pinpoint -->
<dependency>
    <groupId>software.amazon.awssdk</groupId>
    <artifactId>pinpoint</artifactId>
    <version>2.16.3</version>
</dependency>
```

If you want, I can switch to the `ses` dependency; the SesClient is a bit different and SesClientV2 appears a bit more modern with refinements example, an unified way to handle raw messages and simple messages. Let me know @eddumelendez

Reply:
> AFAIK, sesv2 allow integration with pinpoint service and ses with simple email service

that comment is raised because of [this](https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-sesv2/src/main/java/com/amazonaws/services/simpleemailv2/AmazonSimpleEmailServiceV2Client.java)

I know pinpoint is another service and has its own sdk but from the link below and for what I took a look via sesv2 api you can use few resources from pinpoint. I will check more later.

Reply:
Thanks for pointing it out. I will hold on to the `ses2` for now. Agreed AWS got this mixed up a bit, maybe Pinpoint is the way forward :-) The links in the javadoc point to different paces. I'll switch to `ses` for now; although my integration tests ran perfectly fine with `ses2`. Good catch @eddumelendez

Reply:
EDIT: @eddumelendez Typo in my earlier comment. Switching to `ses` for now.